pallets/rc-migrator/src/accounts.rs

```diff
@@ -298,18 +329,36 @@ impl<T: Config> Pallet<T> {
 			pallet_balances::Freezes::<T>::get(&who).into();
 
 		for freeze in &freezes {
-			<T as Config>::Currency::thaw(&freeze.id, &who)
-				// TODO: handle error
-				.unwrap();
+			if let Err(e) = <T as Config>::Currency::thaw(&freeze.id, &who) {
```

Comment:
```suggestion
			if let Err(e) = <T as Config>::Currency::thaw(&freeze.id, &who).defensive() {
```
Then it also fails the test directly.

Reply:
there is actually defensive one level up if this function return an error